hasura.planx.uk/migrations/1681897578801_alter_table_public_payment_requests_add_column_govpay_payment_id/up.sql

```diff
@@ -0,0 +1,4 @@
+alter table "public"."payment_requests" add column "govpay_payment_id" text
+ null unique;
+
+comment on column "public"."payment_requests"."govpay_payment_id" is E'ID returned by GovPay when payment is initialised';
```

Comment:
:+1: thanks for comment here

i noticed auto-generated Hasura changelog comment on this PR reads `lowcal_sessions` not `payment_requests` which is a bit strange - maybe just not getting properly updated?

Reply:
Yeah I think you're right - it's not picking up this change, but is picking up the change to `lowcal_sessions` made in `tables.yml`.

Something that's worth being aware of in future PRs!